torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -421,13 +421,13 @@ std::string dump_nccl_trace(
     printNcclCommProxyTrace("Received dump signal " + ncclUniqueIDStr, dump);
   }
 #endif // defined(USE_ROCM) && defined(NCCL_COMM_DUMP)
-  return FlightRecorder::get()->dump(
+  return FlightRecorder<at::cuda::CUDAEvent>::get()->dump(
```

Comment:
nit: `using FlightRecorderCUDA = FlightRecorder<at::cuda::CUDAEvent>;`
or `typedef ...`
to reduce the verbosity.